resources/views/components/form-input.blade.php

```diff
@@ -0,0 +1,12 @@
+<div>
+    <label for="{{ $id }}" class="block font-medium text-gray-700">
+        {{ $name }}
+    </label>
+    <div class="mt-1">
+        <input type="{{ $type }}" name="{{ $id }}" id="{{ $id }}"
```

Comment:
```suggestion
        <input type="{{ $type }}" name="{{ $field }}" id="{{ $id }}"
```
I would add more readable variable for name attribute.